### PR TITLE
Feature: Send image with text via composer

### LINKED ISSUE #2258: [HOLD for payment June 29] Update compose box to send photos and text in one message
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/master/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Expected Result:**
1. When uploading image attachments in Expensify.cash, the [existing attachment preview modal](https://github.com/Expensify/Expensify.cash/blob/ed8b81e78ad3a8c031ad02338e56990cc0c06f3b/src/components/AttachmentModal.js) should appear. Instead of [saying "Upload"](https://github.com/Expensify/Expensify.cash/blob/ed8b81e78ad3a8c031ad02338e56990cc0c06f3b/src/components/AttachmentModal.js#L132), it should now say "Send". 

This should work the same for all three modes of interaction:
    - Manually picking an image using the [AttachmentPicker](https://github.com/Expensify/Expensify.cash/tree/ed8b81e78ad3a8c031ad02338e56990cc0c06f3b/src/components/AttachmentPicker),
    - [Drag-and-dropping a photo into the `ReportActionCompose`](https://github.com/Expensify/Expensify.cash/blob/2336d7bcea21f90044a888c6a804f85f4de2a57c/src/pages/home/report/ReportActionCompose.js#L403-L413),
    - [Pasting a photo into the `ReportActionCompose`](https://github.com/Expensify/Expensify.cash/blob/2336d7bcea21f90044a888c6a804f85f4de2a57c/src/pages/home/report/ReportActionCompose.js#L419) from the platform's native clipboard.

2. After hitting "Send", any draft text in the composition box should be sent, followed by the attached image below the text.

@mallenexpensify @shawnborton if you have nothing else to add here then feel free to apply the `External` label so we can get this on Upwork.

**Actual Result:**
- Image uploads and text in compose box persists. 

## Action Performed:

**Web/Desktop**
- Type text in compose box
- Drag/drop image to compose box to upload 
- Click 'Upload`

**Mobile web/app**
- Type text in compose box
- Tap attachment/paperclip icon
- Tap `Photo Library` or `Choose from Gallery`
- Select photo and tap `Choose` or `Upload` to upload

## Platform:
Where is this issue occurring?

- [x] Web 
- [x]  iOS
- [x]  Android
- [x]  Desktop App
- [x]  Mobile Web

**Version Number:**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:** https://github.com/Expensify/Expensify/issues/156214
**Platform - version:**  All

**Notes/Photos/Videos:** 
Big ol' thread in Slack - [link](https://expensify.slack.com/archives/C011W8BJ9L6/p1611454781167100)
iOS/Messenger is what we're wanting to mimic 
IMG_5636

IMG_5637

**Slack does a good job of handling this but they put they post the text above the image.** 
image

### PR DESCRIPTION
**Pull request by parasharrajat**
### Fixed Issues

Fixes #2258

### Tests | QA Steps
1. Open E.cash.
2. Open any conversation.
3. Type any message in the composer but don't send it.
4. Add an attachment from the plus menu.
5. Button should say send and After clicking send text should be posted followed by that image.
6. Repeat all the above steps in each platform.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

#### Web

https://user-images.githubusercontent.com/24370807/122802126-9dfd5800-d2e2-11eb-9a94-ad43b555c4b0.mp4

#### Mobile Web

https://user-images.githubusercontent.com/24370807/122808422-81651e00-d2ea-11eb-86d1-02ba33b6ff63.mp4

#### Desktop

https://user-images.githubusercontent.com/24370807/122802143-a190df00-d2e2-11eb-81a9-c3d48076df26.mp4

#### iOS

https://user-images.githubusercontent.com/24370807/122808446-87f39580-d2ea-11eb-83f4-2c268010d8c4.mp4

#### Android

https://user-images.githubusercontent.com/24370807/122802148-a48bcf80-d2e2-11eb-9b4c-3591c6ad0428.mp4

### CODE REVIEW

File: src/components/AttachmentModal.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ class AttachmentModal extends PureComponent {
                             success
                             style={[styles.buttonConfirm]}
                             textStyles={[styles.buttonConfirmText]}
-                            text={this.props.translate('common.upload')}
+                            text={this.props.translate('common.send')}
```
Comment (Jag96) [inline src/components/AttachmentModal.js:140]: Can you remove the old `upload` const value if it is unused now?

File: src/languages/en.js [hunk 2]
```diff
@@ -49,6 +49,7 @@ export default {
         invite: 'Invite',
         iAcceptThe: 'I accept the ',
         passwordCannotBeBlank: 'Password cannot be blank',
+        send: 'Send',
```
Comment (Jag96) [inline src/languages/en.js:52]: Can you also make sure these updates are included in the `es.js` file? You can just google translate to Spanish for now

**Review by Jag96 — CHANGES_REQUESTED** [review]
Tested and works well! Just a couple of requests for the language files

**Comment by parasharrajat** [comment]
@Jag96 Updated.

**Comment by parasharrajat** [comment]
Ready for review

**Review by Jag96 — APPROVED** [review]
LGTM

**Pull request merged**